.github/workflows/publish_pypi.yml

```diff
@@ -3,6 +3,9 @@ name: Build and publish package to PyPI
 on:
```

Comment:
Should we add a badge for this in the README?

Reply:
Should be a good idea. Alternatively or in addition we can use one of those actions to open an issue if one of the jobs fails

Reply:
Works now - https://github.com/Saransh-cpp/PyBaMM/issues/13!